pkg/controller/apikey/index.go

```diff
@@ -42,16 +42,11 @@ func (c *Controller) HandleIndex() http.Handler {
 			controller.Unauthorized(w, r, c.h)
 			return
 		}
-		if !membership.Can(rbac.APIKeyRead) {
-			controller.Unauthorized(w, r, c.h)
-			return
-		}
-
 		currentRealm := membership.Realm
 
 		pageParams, err := pagination.FromRequest(r)
 		if err != nil {
-			controller.InternalError(w, r, c.h, err)
```

Comment:
Twas bug. Would have paged if someone edited query params. Changed to user error.

Reply:
good call. Did we look for other instances of pagination.FromRequest doing this? might be common

Reply:
Pulled this out into https://github.com/google/exposure-notifications-verification-server/pull/1438. I can rebase if that makes it in first.